content/pytorch/concepts/tensor-operations/terms/index_select/index_select.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: '.index_select()'
+Description: 'Extracts specific elements from a tensor to return a new tensor.'
+Subjects:
+  - 'AI'
+  - 'Data Science'
+Tags:
+  - 'AI'
+  - 'Arrays'
+  - 'Data Structures'
+  - 'Deep Learning'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'paths/computer-science'
+---
+
+**Index_select()** is used to extract specific elements (rows, columns) from the tensor to return a new tensor. 
+
+## Syntax
+
+```pseudo
+torch.index_select(input,dim,index,out)
```

Comment:
```suggestion
torch.index_select(input, dim, index, out)
```
It would be helpful to add a note below regarding the syntax: 'If the result is to be stored in an existing tensor, the output tensor can be specified using the optional out argument.'